src/app/database/pouch-database.ts

```diff
@@ -107,7 +107,7 @@ export class PouchDatabase extends Database {
 
 
   private notifyError(err) {
-    this.alertService.addWarning(err.message + '(' + err.id + ')');
+    this.alertService.addWarning(err.message + ' (' + err.status + ')', true);
```

Comment:
If we silence the error reporting here, we should make sure that higher level modules consistently do report what fails. But I guess that's the whole point of this PR anyway.